Cargo.toml

```diff
@@ -29,16 +33,18 @@ iovec = "0.1"
 itoa = "0.4.1"
 log = "0.4"
 net2 = { version = "0.2.32", optional = true }
+pin-utils = "0.1.0-alpha.4"
 time = "0.1"
-tokio = { version = "0.1.14", optional = true, default-features = false, features = ["rt-full"] }
+tokio = { path = "../tokio/tokio", optional = true, default-features = false, features = ["rt-full"] }
 tokio-buf = "0.1"
-tokio-executor = { version = "0.1.0", optional = true }
-tokio-io = "0.1"
-tokio-reactor = { version = "0.1", optional = true }
-tokio-tcp = { version = "0.1", optional = true }
-tokio-threadpool = { version = "0.1.3", optional = true }
-tokio-timer = { version = "0.2", optional = true }
-want = "0.0.6"
+tokio-executor = { path = "../tokio/tokio-executor", optional = true }
```

Comment:
might wanna make these git deps so I can run this locally? 😄

Reply:
Oh right, haven't updated since it was merged to master XD